karyon2-admin/build.gradle

```diff
@@ -18,8 +18,8 @@ apply plugin: 'osgi'
 dependencies {
     dependencies {
         compile 'javax.ws.rs:jsr311-api:1.1.1'
-        compile 'javax.servlet:servlet-api:2.5'
-        compile 'org.mortbay.jetty:jetty:6.1.26'
+        provided 'javax.servlet:javax.servlet-api:3.0.1'
+        provided 'org.mortbay.jetty:jetty:6.1.26'
```

Comment:
What was the specific issue that prompted this PR?   Also, this is a very old version of jetty.  Perhaps we should consider changing this to the latest version of jetty in the org.eclipse.jetty org.